### ISSUE TITLE
dog output shows STDOUT and STDERR in realtime

### ISSUE THREAD

**Issue by xsb** [xsb/dog#6]
When launching a task, both STDOUT and STDERR are streamed and printed to the console output.

**Comment by xsb** [comment]
Fixed by #15

**Comment by xsb** [comment]
I'm reopening this issue, as this doesn't work now.

Test with:

```yml
- task: long-run
  description: A task that takes 5 seconds
  run: |
    for i in $(seq 1 5); do
      sleep 1
      echo "> $i/5"
    done
```

**Comment by xsb** [comment]
Closed by #48
